clients/c/src/orpClient.c

```diff
@@ -302,7 +328,44 @@ static void orp_Dispatch
     struct orp_Message *message
 )
 {
-    // nothing implemented yet
+    switch(message->type)
+    {
+        case ORP_PACKET_TYPE_UNKNOWN:
+        case ORP_RQST_INPUT_CREATE:
+        case ORP_RESP_INPUT_CREATE:
+        case ORP_RQST_OUTPUT_CREATE:
+        case ORP_RESP_OUTPUT_CREATE:
+        case ORP_RQST_DELETE:
+        case ORP_RESP_DELETE:
+        case ORP_RQST_HANDLER_ADD:
+        case ORP_RESP_HANDLER_ADD:
+        case ORP_RQST_HANDLER_REM:
+        case ORP_RESP_HANDLER_REM:
+        case ORP_RQST_PUSH:
+        case ORP_RESP_PUSH:
+        case ORP_RQST_GET:
+        case ORP_RESP_GET:
+        case ORP_RQST_EXAMPLE_SET:
+        case ORP_RESP_EXAMPLE_SET:
+        case ORP_RQST_SENSOR_CREATE:
+        case ORP_RESP_SENSOR_CREATE:
+        case ORP_RQST_SENSOR_REMOVE:
+        case ORP_RESP_SENSOR_REMOVE:
+        case ORP_NTFY_HANDLER_CALL:
+        case ORP_RESP_HANDLER_CALL:
+        case ORP_NTFY_SENSOR_CALL:
+        case ORP_RESP_SENSOR_CALL:
+            break;
+
+        case ORP_SYNC_SYN:
+            (void)orp_Respond(ORP_SYNC_SYNACK, );
+            break;
+
+        case ORP_SYNC_SYNACK:
+        case ORP_SYNC_ACK:
+        case ORP_RESP_UNKNOWN_RQST:
+        default:
```

Comment:
You could add a break here.

From The C programming language - Second edition (K&R 2):

Chapter 3.4 Switch

_As a matter of good form, put a break after the last case (the default here) even though it's logically unnecessary. Some day when another case gets added at the end, this bit of defensive programming will save you._